evals/datasets_/arc.py

```diff
@@ -45,27 +50,36 @@ def load_uhura_arc_easy(language_bcp_47, nr):
         ds = _load_dataset(slug_uhura_arc_easy, tags_uhura_arc_easy[language_bcp_47])
         ds = ds.map(add_choices)
         ds = ds.rename_column("answerKey", "answer")
-        train_ids = common_ids_train[nr:nr+3]
-        examples = ds["train"].filter(lambda x: x["id"] in train_ids)
         task = ds["test"].filter(lambda x: x["id"] == common_ids_test[nr])[0]
-        return "masakhane/uhura-arc-easy", examples, task
+        return "masakhane/uhura-arc-easy", task, "human"
     if language_bcp_47 in tags_uhura_arc_easy_translated.keys():
-        ds = _load_dataset(slug_uhura_arc_easy_translated, tags_uhura_arc_easy_translated[language_bcp_47])
+        ds = _load_dataset(
+            slug_uhura_arc_easy_translated,
+            tags_uhura_arc_easy_translated[language_bcp_47],
+        )
         ds = ds.rename_column("answerKey", "answer")
-        train_ids = common_ids_train[nr:nr+3]
-        examples = ds["train"].filter(lambda x: x["id"] in train_ids)
-        # raise Exception(language_bcp_47)
         task = ds["test"].filter(lambda x: x["id"] == common_ids_test[nr])[0]
-        return "fair-forward/arc-easy-autotranslated", examples, task
+        return "fair-forward/arc-easy-autotranslated", task, "machine"
     else:
         return None, None, None
 
+
+def load_uhura_arc_challenge(language_bcp_47, nr):
+    ds_name = "jlahd/uhura_arc_challenge"
+    if language_bcp_47 in _get_dataset_config_names(ds_name):
+        ds = _load_dataset(ds_name, language_bcp_47)
+        task = ds["test"][nr]
+        return ds_name, task
```

Comment:
would be cool to have the challenge dataset rather than the easy one, but I can't find this one on Huggingface or elsewhere, maybe it's a hallucination?